auto_round/modelling/__init__.py

```diff
@@ -11,3 +11,6 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+from .gpt_oss import *
```

Comment:
If Transformers changes a module path (e.g., transformers.models.llama4.modeling_llama4), it will cause an import error. Could we instead import the corresponding file from the modeling folder on demand?

Reply:
for example, some legacy transforms may have no llama4